src/classes/AccountsService.cls

```diff
@@ -0,0 +1,114 @@
+public class AccountsService {
+    
+    // Unit of Work
+    private static fflib_SObjectUnitOfWork uow = new fflib_SObjectUnitOfWork(
+        new List<SObjectType> { Account.SObjectType  }
+    );
+    
+    public static List<Account> get100AccountRecords(String selectedBillingCity, String selectedBillingState, String selectedBillingCountry)
+    {
+        List<Account> accountsList = new AccountsSelector(true).selectByAccountWithFilters(selectedBillingCity, selectedBillingState, selectedBillingCountry);
+        return accountsList;
+    }
+    
+    public static List<Account> getAccountRecordsWithFilter(String selectedBillingCity, String selectedBillingState, String selectedBillingCountry)
+    {
+        List<Account> accountsList = new AccountsSelector(true).selectAccountWithFilter(selectedBillingCity, selectedBillingState, selectedBillingCountry);
+        return accountsList;
+    }
+    
+    public static void manageContactIDs(id mycontactid)
+    {
+        
+        List<Account> accountsList = new List <Account>();
+        List<Contact> contactList = new ContactsSelector(true).selectById(new Set<Id> {mycontactid});
+        //system.debug('account id = '+ contactList);
+        if(contactList.size()==0)
```

Comment:
remove this if condition. It has no use, you can check condition `contactList.size() != 0` with below condition.